pkgs/tools/misc/octofetch/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, openssl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "octofetch";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "azur1s";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-ciBFTVdHotjifNAoXJjI9CumyK98OkMmfWMbrEldlNI=";
+  };
+
+  cargoSha256 = "sha256-Gzemm5HY6YwlxesQlil6R+34OtAeU2k7f/9+Lll3i8k=";
+
+  preBuild = ''
+    export OPENSSL_DIR=${openssl.dev}
+    export OPENSSL_LIB_DIR=${openssl.out}/lib
+  '';
```

This is an alternate way to specify the dependencies. However , I think sandro was trying to imply:
```suggestion
  nativeBuildInputs = [ pkg-config ];

  buildInputs = [ openssl ];
```